luigi/contrib/bigquery.py

```diff
@@ -326,14 +326,14 @@ def run_job(self, project_id, body, dataset=None):
         job_id = new_job['jobReference']['jobId']
         logger.info('Started import job %s:%s', project_id, job_id)
         while True:
-            status = self.client.jobs().get(projectId=project_id, jobId=job_id).execute()
+            status = self.client.jobs().get(projectId=project_id, jobId=job_id).execute(num_retries=10)
             if status['status']['state'] == 'DONE':
-                if status['status'].get('errors'):
-                    raise Exception('BigQuery job failed: {}'.format(status['status']['errors']))
+                if status['status'].get('errorResult'):
+                    raise Exception('BigQuery job failed: {}'.format(status['status']['errorResult']))
```

Comment:
Is `'errors'` no more in the dict at this point? I would try to make sure that the exception dumps any information available. Why not all the `status`?